auth/src/api/services/key-service.ts

```diff
@@ -1,32 +1,40 @@
-import { Keys } from '../../config';
+import { Config } from '../../config';
 import { KeyHandler } from '../interfaces/key-handler';
 
+const AUTH_DEV_KEY = 'auth-dev-key';
+
 export class KeyService extends KeyHandler {
-    protected privateTokenKey: string;
-    protected privateCookieKey: string;
-    protected publicTokenKey: string;
-    protected publicCookieKey: string;
+    protected tokenKey = AUTH_DEV_KEY;
+    protected cookieKey = AUTH_DEV_KEY;
 
     public constructor() {
         super();
 
         // Load key files early to detect missing ones
-        this.privateTokenKey = Keys.privateTokenKey();
-        this.privateCookieKey = Keys.privateCookieKey();
-        this.publicTokenKey = Keys.publicTokenKey();
-        this.publicCookieKey = Keys.publicCookieKey();
+        this.loadKeys();
     }
 
-    public getPrivateTokenKey(): string {
-        return this.privateTokenKey;
-    }
-    public getPrivateCookieKey(): string {
-        return this.privateCookieKey;
+    public getCookieKey(): string {
+        return this.cookieKey;
     }
-    public getPublicTokenKey(): string {
-        return this.publicTokenKey;
+
+    public getTokenKey(): string {
+        return this.tokenKey;
     }
-    public getPublicCookieKey(): string {
-        return this.publicCookieKey;
+
+    private loadKeys(): void {
+        if (Config.isInDevMode()) {
```

Comment:
Also load in dev mode. Put these keys into the dockerfile.dev (ENV AUTH_TOKEN_KEY auth-dev-key, ...)